hashicorp/boundary-ui
# Automate copyright headers addition

# Description

This PR automates the addition of copyright headers as discussed at Tech time.

## How to Test

I run the test procedure described below within [this dummy PR](https://github.com/hashicorp/boundary-ui/pull/2441), you can see [the automated commit added here](https://github.com/hashicorp/boundary-ui/pull/2441/commits/fe0ee7c1287e5b5f60b9acd3f87d141a7269f806).

- Create a new branch checking out from this PR.
- Add a JS file within the codebase. i.e: `boundary-ui/ui/admin/app/random.js`.
- Push the branch and start a PR.
- Review your PR and make sure: There is an automated commit that add headers to the `random.js` file.
